src/integrationTest/java/uk/gov/hmcts/darts/event/controller/DarNotifyControllerTest.java

```diff
@@ -56,14 +68,50 @@ void setup() {
 
     @Test
     void shouldSendDarNotifyEventSoapAction() throws Exception {
-        darPcStub.respondWithSuccessResponse();
+        darPcStub.respondWithSuccessResponse(OffsetDateTime.now(clock));
+
+        mockMvc.perform(post("/events/dar-notify")
+                            .contentType(APPLICATION_JSON_VALUE)
+                            .content(VALID_NOTIFICATION_JSON))
+            .andExpect(status().is2xxSuccessful());
+
+        darPcStub.verifyNotificationReceivedWithBody(EXPECTED_DAR_PC_NOTIFICATION);
```

Comment:
Do you need a verification test here to make sure the message isn't printed when inside the 90 second threshold?

Reply:
I verified this manually but it would be worth automated verifications so will stick this in good spot :)